packaging/ondemand-systemstatus.spec

```diff
@@ -56,11 +61,17 @@ scl enable ondemand - << \EOS
 export PASSENGER_APP_ENV=production
 export PASSENGER_BASE_URI=/pun/sys/%{app_name}
 %endif
+export GEM_HOME=$(pwd)/gems-build
+export GEM_PATH=$(pwd)/gems-build:$GEM_PATH
```

Comment:
Why do we set GEM_HOME to a build directory and then move the gems to the final location, versus just setting GEM_HOME to `%{buildroot}%{gem_home}`?

Reply:
See my comment here: https://github.com/OSC/ondemand/pull/87#issuecomment-547105885

The first thing %install does is remove %buildroot, so that won't work.